gnuradio-runtime/include/gnuradio/nco.h

```diff
@@ -49,10 +49,10 @@ class nco
         phase += phase_inc;
```

Comment:
Would you please combine `step(void)` and `step(int)` into just `step (int n = 1)` ... silly to have 2 functions doing the same thing for all practical purposes. The extra `*n` will cost an extra CPU cycle or 2, but I strongly doubt it will make or break anybody's flowgraph functionality!